src/plot.rs

```diff
@@ -87,24 +87,11 @@ pub fn plot_total_energy(
             },
         )
         .expect("Should be able to estimate the text size");
-    let x_label_size = root_drawing_area
-        .estimate_text_size(
-            &format_label(&y_max_total_energy),
-            &TextStyle {
-                font: ("Sans-serif", 15).into_font(),
```

Comment:
With this change the labels look funny. I noticed that one was using 15px font size, and the other was using 20px font size. Making them into just one 20px size looks wrong, so I'll undo this change.